src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6163,4 +6163,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_UnexpectedArgumentList" xml:space="preserve">
     <value>Unexpected argument list.</value>
   </data>
+  <data name="ERR_UnexpectedOrMissingConstructorInitializerInRecord" xml:space="preserve">
+    <value>A constructor declared in a record with parameters must have 'this' constructor initializer.</value>
```

Comment:
>a record with parameters [](start = 37, length = 24)

nit: we'll want to align on terminology. Andy has been using "positional records" for such records. I lean towards "record with positional members".

Reply:
> nit: we'll want to align on terminology. Andy has been using "positional records" for such records. I lean towards "record with positional members".

Agree, at some point we should do that.

---
In reply to: [434399626](https://github.com/dotnet/roslyn/pull/44803#discussion_r434399626) [](ancestors = 434399626)

Reply:
Filed [https://github.com/dotnet/roslyn/issues/44826](https://github.com/dotnet/roslyn/issues/44826)

---
In reply to: [434694065](https://github.com/dotnet/roslyn/pull/44803#discussion_r434694065) [](ancestors = 434694065,434399626)